surfinBH/_fit_evaluators/fit_7dq4Emri.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
+from surfinBH import surfinBH
+import warnings
+import sys
+
+#=============================================================================
+class Fit7dq4Emri(surfinBH.SurFinBH):
+    """ A class for the NRSur7dq4EmriRemnant model presented in Boschini et al.,
+    arxiv:[TO_BE_CHANGED] (hereafter THE PAPER). 
+    
+    This model predicts the final mass mf and final spin chif, for the remnants 
+    of precessing binary black hole systems. 
```

Comment:
black hole systems. --> "black hole systems, extending to arbitrarily large mass ratios."